test/boost/bti_key_translation_test.cc

```diff
@@ -303,7 +303,9 @@ BOOST_AUTO_TEST_CASE(test_lcb_mismatch) {
                     fmt_fragmented_buffer{fragments_a},
                     fmt_fragmented_buffer{fragments_b});
                 auto generator_from_range = [&](auto& r) -> std::generator<std::ranges::range_value_t<decltype(r)>> {
-                    co_yield std::ranges::elements_of(r);
+                    for (auto e : r) {
+                        co_yield e;
+                    }
```

Comment:
What's wrong with the original code? It looks like it's correct.

I don't mind the change, just curious.

Reply:
Type mismatch. `std::ranges::elements_of(r)` generates references, not values. But this generator wants to return values, not references.

Reply:
(And it makes perfect sense.)

Reply:
https://lists.isocpp.org/std-proposals/2025/08/15009.php